system-int-test/test/cmr/system_int_test/search/variable/concept_retrieval_test.clj

```diff
@@ -0,0 +1,109 @@
+(ns cmr.system-int-test.search.variable.concept-retrieval-test
+  "Integration test for variable retrieval via the /concepts/:concept-id and
+  /concepts/:concept-id/:revision-id endpoints."
+  (:require
+   [cheshire.core :as json]
```

Comment:
yeah, it was a copy paste from collections concept retrieval test. We're going to be adding a bunch of tests here, though ... I guess I can remove them (we'll just be adding them back).